winui/Common/compact-sizing.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Right to left in WinUI Controls | Syncfusion
+description: Learn here about Right to left FlowDirection support in Syncfusion WinUI Project Reunion controls and more details. 
+platform: WinUI
+control: RightToLeft
+documentation: ug
+---
+
+# Compact Sizing in WinUI Controls
+
+There are two types of spacing support for controls in WinUI.
+1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input
```

Comment:
1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input.

Reply:
updated the changes